pokemongo_bot/cell_workers/recycle_items.py

```diff
@@ -29,6 +29,9 @@ def work(self):
                 else:
                     logger.log("-- Failed to discard " + item_name, 'red')
 
+        if not self.bot.has_space_for_loot:
+            logger.log("-- Inventory is still lacking space after recycling.. You might want to fix your config to recycle more things.", 'yellow')
```

Comment:
I don't think we need the `-- ` and you probably don't want the double period between your sentences.